components/efuse/src/esp_efuse_utility.c

```diff
@@ -452,11 +461,29 @@ bool esp_efuse_utility_load_efuses_from_flash(void)
             return true;
         }
     }
+
+    ESP_EARLY_LOGW(TAG, "no fuses found in flash");
+
     return false;
 }
 
+void write_efuses_to_ram(void)
+{
+    ESP_EARLY_LOGW(TAG, "Writing fuses to ram");
+
+    memset(virt_blocks, 0, sizeof(virt_blocks));
+
+    esp_efuse_utility_debug_dump_blocks();
+
+    ESP_EARLY_LOGW(TAG, "Write complete");
+}
+
 void esp_efuse_utility_write_efuses_to_flash(void)
 {
+    ESP_EARLY_LOGW(TAG, "Writing fuses to flash: 0x%x (offset)", esp_efuse_flash_offset);
+
+    esp_efuse_utility_debug_dump_blocks();
```

Comment:
At some point, for example in esp_efuse_init_virtual_mode_in_flash(), we will see two dumps, with the same data.
If we need to print dump after burning or at startup we can directly call esp_efuse_utility_debug_dump_blocks, 
So I suggest to do not putting it here, to not be overloaded by the logs.

Reply:
If you are concerned about too many logs (reasonable), perhaps we can put this in a kConfig? a `EFUSE_VIRTUAL_LOG_ALL_WRITES`?

Virtual eFuse is by definition a _debug_ mode. Users _need_ an easy way to see which fuses would be burned, before burning the real fuses. 

Adding a bunch of `esp_efuse_utility_debug_dump_blocks`() calls to your own code is cumbersome.